templates/crate/source.html

```diff
@@ -113,21 +113,14 @@
             {# If the file has content, then display it in a codeblock #}
             {%- if file_content -%}
                 <div id="source-code" class="pure-u-1 pure-u-sm-17-24 pure-u-md-19-24">
-                    <pre><code>{{ file_content }}</code></pre>
+                    {{- file_content | highlight(lang=file.name) -}}
```

Comment:
the argument to `highlight` is still called `lang`. 

Would it still work with a pure lang name / file extension? Or do we _need_ a full filename?

Reply:
It works with either. It will first try lookup via the full string, then if it can extract an extension it will try that.